projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -450,6 +450,6 @@ export class TuiInputDateTimeComponent
             return tuiDay;
         }
 
-        return [tuiDay, tuiTime];
+        return value as [TuiDay, TuiTime];
```

Comment:
The same as `main`-branch:
https://github.com/taiga-family/taiga-ui/blob/daeba7b6363d9312ab87d2fc1eab16541c4ecce0/projects/legacy/components/input-date-time/input-date-time.component.ts#L441